primitives/manta/src/types.rs

```diff
@@ -14,42 +14,56 @@
 // You should have received a copy of the GNU General Public License
 // along with Manta.  If not, see <http://www.gnu.org/licenses/>.
 
-use sp_runtime::traits::{BlakeTwo256, IdentifyAccount, Verify};
+//! Types
 
-pub use manta_accounting::asset::{AssetIdType, AssetValueType};
+use sp_core::H256;
+use sp_runtime::{
+    generic,
+    traits::{BlakeTwo256, IdentifyAccount, Verify},
+    MultiSignature, OpaqueExtrinsic,
+};
 
-/// An index to a block.
+/// Block Number Type
 pub type BlockNumber = u32;
 
 /// Alias to 512-bit hash when used in the context of a transaction signature on the chain.
-pub type Signature = sp_runtime::MultiSignature;
+pub type Signature = MultiSignature;
+
+/// Signer Type
+pub type Signer = <Signature as Verify>::Signer;
 
 /// Some way of identifying an account on the chain. We intentionally make it equivalent
 /// to the public key of our transaction signing scheme.
-pub type AccountId = <<Signature as Verify>::Signer as IdentifyAccount>::AccountId;
+pub type AccountId = <Signer as IdentifyAccount>::AccountId;
 
-/// The type for looking up accounts. We don't expect more than 4 billion of them, but you
-/// never know...
+/// Account Index Type
+///
+/// This index is used to look up accounts.
 pub type AccountIndex = u32;
 
-/// Balance of an account.
-pub type Balance = AssetValueType;
+/// Calamari Asset Id Type
+pub type CalamariAssetId = u128;
+
+/// Dolphin Asset Id Type
+pub type DolphinAssetId = u32;
```

Comment:
Though you're saying "we won't need u128 asset ID until we restart it", we're gonna restart it next week, so although we don't need the migration for dolphin, dolphin should use the new u128 asset ID as well, no?

Reply:
https://github.com/Manta-Network/Manta/issues/883